### ISSUE TITLE
Installation failure and panic on golang 1.17

### ISSUE THREAD

**Issue by adamcohen2** [tetafro/godot#26]
I've tried following the docs to install `godot` with `go 1.17.1` using `go get -u github.com/tetafro/godot/cmd/godot`, however, that gives me a failure:

```shell
docker run -it --rm golang:1.17.1 /bin/bash

root@9cdd0824ea9c:/go# go get -u github.com/tetafro/godot/cmd/godot
go: downloading github.com/tetafro/godot v1.4.11
go: downloading github.com/tetafro/godot/cmd/godot v0.0.0-20210115153532-96ff7da53d7b
go: downloading github.com/tetafro/godot v1.4.0
github.com/tetafro/godot/cmd/godot: ambiguous import: found package github.com/tetafro/godot/cmd/godot in multiple modules:
	github.com/tetafro/godot v1.4.11 (/go/pkg/mod/github.com/tetafro/godot@v1.4.11/cmd/godot)
	github.com/tetafro/godot/cmd/godot v0.0.0-20210115153532-96ff7da53d7b (/go/pkg/mod/github.com/tetafro/godot/cmd/godot@v0.0.0-20210115153532-96ff7da53d7b)
```

Since installing executables using `go get` is [now deprecated](https://golang.org/doc/go-get-install-deprecation) as of `go 1.17`, I've tried using `go install github.com/tetafro/godot/cmd/godot@latest`, but that produces a binary that panics on certain files:

```shell
docker run -it --rm golang:1.17.1 /bin/bash

root@63a6339db0ae:/go# echo -e "package test \n func test() {}" >> /testing.go

root@63a6339db0ae:/go# go install github.com/tetafro/godot/cmd/godot@latest

go: downloading github.com/tetafro/godot v1.4.11
go: downloading github.com/tetafro/godot/cmd/godot v0.0.0-20210115153532-96ff7da53d7b
go: downloading github.com/tetafro/godot v1.4.0

root@63a6339db0ae:/go# godot /testing.go

panic: runtime error: index out of range [-1]

goroutine 1 [running]:
github.com/tetafro/godot.newParsedFile(0xc0000d2000, 0xc0000b8080)
	/go/pkg/mod/github.com/tetafro/godot@v1.4.0/getters.go:41 +0x2ab
github.com/tetafro/godot.Run(0x4e39c0, 0x0, {{0x4ffe32, 0xc}, {0x0, 0x0, 0x0}, 0x1, 0x0})
	/go/pkg/mod/github.com/tetafro/godot@v1.4.0/godot.go:46 +0x65
main.main()
	/go/pkg/mod/github.com/tetafro/godot/cmd/godot@v0.0.0-20210115153532-96ff7da53d7b/main.go:95 +0x60e
```

I think the documentation should be updated to explain how to use `godot` with `go > 1.17`

**Comment by tetafro** [comment]
Hi. Thanks for detailed report and PR.

I'm trying to figure out how to fix it without having installation hacks in readme. Here is a great [explanation](https://github.com/golang/go/issues/39007#issuecomment-924957208) of the problem, if you're interested.

**Comment by adamcohen2** [comment]
> You can actually try it right now. I think it's fixed.

I just tried, works for me now. Looks like you must have fixed it through the `v0.0.0-retracted` tag. Thanks for the quick fix, closing this issue and the related PR!
